pennylane/qchem/convert.py

```diff
@@ -316,7 +317,7 @@ def import_operator(qubit_observable, format="openfermion", wires=None, tol=1e01
             If ``None``, the identity map (e.g., ``0->0, 1->1, ...``) will be used.
         tol (float): Tolerance in `machine epsilon <https://numpy.org/doc/stable/reference/generated/numpy.real_if_close.html>`_
             for the imaginary part of the coefficients in ``qubit_observable``.
-            Coefficients with imaginary part less than 2.22e-16*tol are considered to be real.
+            Coefficients with imaginary part less than :math:`2.22e-16 \cdot \text{tol}` are considered to be real.
```

Comment:
```suggestion
            Coefficients with imaginary part less than :math:`2.22e^{-16} \cdot \text{tol}` are considered to be real.
```

Reply:
For exponentiation

Reply:
I think that would look like "e" to the power of -16, which is not proper scientific notation. I'll convert it to `2.22 \cdot 10^{-16}`.

Reply:
💯